tftest.py

```diff
@@ -0,0 +1,14 @@
+import gensim,os,time 
```

Comment:
If it's generally useful (to other people), it should go to `scripts`. Otherwise please remove. @tmylk